UPGRADE-1.9.md

```diff
@@ -76,6 +48,52 @@
     +           permanent: false
     ```
 
+#### Upgrades Symfony to v5.2
+
+1. Upgrade Symfony dependencies by:
+
+    ```bash
+    composer config extra.symfony.require "^5.2"
+    composer require "symfony/browser-kit" "^5.2"
+    composer require "symfony/debug-bundle" "^5.2"
+    composer require "symfony/intl" "^5.2"
+    composer require "symfony/web-profiler-bundle" "^5.2"
+    ```
+
+1. Add new bundles to your list of used bundles in `config/bundles.php` if they are not already there:
+
+    ```diff
+    +   BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
+    +   SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
```

Comment:
```suggestion
    ```diff
    +   BabDev\PagerfantaBundle\BabDevPagerfantaBundle::class => ['all' => true],
    +   SyliusLabs\Polyfill\Symfony\Security\Bundle\SyliusLabsPolyfillSymfonySecurityBundle::class => ['all' => true],
    ```
```

The code block is opened but not closed after.